### PR TITLE
coins Value component

### PR DESCRIPTION
**Pull request by Demandtech**
fix: #55 
screencapture-localhost-3000-stories-coinvalue-2022-12-12-14_07_25
screencapture-localhost-3000-stories-coinvalue-2022-12-12-14_06_38

### CODE REVIEW

**Comment by Demandtech** [comment]
I get the blue card from the left angle from the latest pull I made from Master;

**Review by mrxshinji — APPROVED** [review]
All good for me

File: src/routes/coin-value/coinValue.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;600;700&family=Raleway:wght@300;400;500;600;700;800;900&display=swap');
+:root {
+  --primary-background: #ffffff;
+  --primary-black-text: #230b34;
+  --primary-dark-grey: #3f3f3f;
+  --primary-grey-text: #626262;
+  --primary-green: #21d91d;
+  --primary-border-radius: 40px;
+}
+.dark {
```
Comment (aftabrehan) [inline src/routes/coin-value/coinValue.css:10]: It's better to have a blank line between every selector.

```suggestion
}

.dark {
```

File: src/routes/coin-value/coinValue.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  --secondary-background: #100d1f;
+  --secondary-white: #ffffff;
+  --secondary-dark-grey: #4f4f4f;
+  --secondary-grey-text: #bdbdbd;
+  --secondary-border-radius: none;
+}
+.coins-container {
```
Comment (aftabrehan) [inline src/routes/coin-value/coinValue.css:17]:
```suggestion
}

.coins-container {
```

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @Demandtech 💪 🚀 💯 

Just a minor adjustment and then it's good to go 🚀

**Pull request merged**